### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by saudiqbal** [acme-dns/acme-dns#350]
I see a lot of pull requests and no activity, is this project abandoned?

**Comment by dusatvoj** [comment]
It looks that @joohoi left the one man show project :confused: . Pretty strange when it's such important project for FOSS community.

**Comment by joohoi** [comment]
Still here. Working on rewriting some tests for the refactoring PR #325
that is a blocker for anything else atm as the main branch will see large
changes.

After the rewrite, there are plans to move it to a different, organization
namespace where the project is able to more conveniently allow adding new
maintainers. As acme-dns operates largely in a trust - space, it will be
interesting to see how to best handle this especially in the light of
recent dependency chain attacks.

I have very limited time at my hands

**Comment by dusatvoj** [comment]
@joohoi , glad to hear that. I'm looking forward for things you mentioned :+1: :pray:

**Comment by saudiqbal** [comment]
Good to see this project is still active, also don't forget about https://github.com/joohoi/acme-dns/issues/345 to make acme-dns more secure.

**Comment by maddes-b** [comment]
> I have very limited time at my hands

Count me in. I would be willing to maintain the current master, and also "port forward" changes to https://github.com/joohoi/acme-dns/pull/325

**Comment by shaneshort** [comment]
@joohoi any update on this? is there anything anyone can do to try and help get this over the line?

**Comment by laf0rge** [comment]
With all respect to @joohoi as the original author, it is clear that there are a lot of users reporting a lot of issues during 2+ years, and PRs are piling up without either getting merged or meaningful feedback. As a maintainer of many other FOSS projects myself, I sympathesize with the overload situation.

But in the interest of the [production] userse of the project, I would certianly appreciate if other maintainer[s] would be enabled to step up. I've read the plan was to do this after #325 - but the last news on that was more than 6 months ago.

Thanks to @maddes-b for offering to step up.

**Comment by joohoi** [comment]
Yeah, I would love to get any and all help possible in #325 to get the tests up-to-par (the main remaining issue atm). That's the main "2.0" PR that needs to be landed in order to get the codebase more maintainable, and it's the mental milestone to get acme-dns to a more distributed maintenance model and moved to acme-dns organization.

**Comment by shaneshort** [comment]
@joohoi I've managed to rope someone in to try and help get this over the line. Is there a discord channel or something we could find you on to chat about it?

**Comment by qzio** [comment]
Any updates on this?
I would really like to use this project, but it's a bit unclear if it is active or not.

Would it make sense to fork and/or rewrite if #325 never gets through?

Maybe there is a similar project that does the same thing?
